integration/csv_fork_test.go

```diff
@@ -95,17 +95,22 @@ func makeTestOutput(r *rpctest.Harness, t *testing.T,
 // them.
 //
 // Overview:
-//  - Pre soft-fork:
-//    - Transactions with non-final lock-times from the PoV of MTP should be
-//      rejected from the mempool.
-//    - Transactions within non-final MTP based lock-times should be accepted
-//      in valid blocks.
 //
-//  - Post soft-fork:
-//    - Transactions with non-final lock-times from the PoV of MTP should be
-//      rejected from the mempool and when found within otherwise valid blocks.
-//    - Transactions with final lock-times from the PoV of MTP should be
-//      accepted to the mempool and mined in future block.
+//   - Pre soft-fork:
```

Comment:
I think these need some manual adjustments to preserve the different levels of indentation.